src/lib/all_type_variant.cpp

```diff
@@ -0,0 +1,19 @@
+#include "all_type_variant.hpp"
+
+#include <cmath>
+
+namespace opossum {
+
+bool all_type_variant_near(const AllTypeVariant& lhs, const AllTypeVariant& rhs, double max_abs_error) {
```

Comment:
There are already similar functions like this in our test suite. This functions seems to me to have a very specific use case and shouldn’t therefore be in `all_type_variant.hpp`. Any chance we can merge this with the other functions in the test suite?

Reply:
By other functions in the test suite you mean EXPECT_NEAR? Merging with it would be more complicated than this function is, among other disadvantages.
This function is tightly bound to AllTypeVariant and should, by my reasoning, be nowhere else than `all_type_variant.hpp`